docs/requirements/ics10-requirements.md

```diff
@@ -1,98 +1,94 @@
-<!-- More detailed information about the requirements engineering process can be found at https://github.com/cosmos/ibc-go/wiki/Requirements-engineering -->
-
 # Business requirements
 
-<!-- They describe why the organization is implementing the product or feature (the benefits the organization hopes to achieve). They provide a reference for making decisions about proposed requirement changes and enhancements (i.e. decide if a proposed requirement is in or out of scope). Business requirements directly influence which user or functional requirements to implement and in what sequence. -->
-
-<!-- Provide a high-level, short description of the software being specified and its purpose, including relevant benefits, objectives, and goals. Relate the software to ecosystem goals or strategies. -->
-
-Using IBC as a mean of commuincating between chains and ecosystems has proven to be useful within Cosmos. There is then value in extending
-this feature into other ecosystems, bringing a battle tested protocol of trusted commuincation as an option to send assets and data.
+Using IBC as a mean of communicating between chains and ecosystems has proven to be useful within Cosmos. There is then value in extending
+this feature into other ecosystems, bringing a battle-tested protocol of trusted communication as an option to send assets and data.
```

Comment:
I would remove 'of trusted communication' as using trust in this context makes me think more of trusted bring, rather than trustworthy